apps/mobile/src/navigation/linking/index.js

```diff
@@ -23,80 +33,148 @@ React Router (web)
+  '/:context(all)/members/:personId':                                     `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Member Profile`,
+  '/:context(all)/:groupSlug/post/:postId':                               `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Post Detail`,
+  '/:context(all)/:groupSlug/map/create':                                 `${AUTH_ROOT_SCREEN_NAME}/Edit Post`,
+  '/:context(all)/topics/:topicName':                                     `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  // TODO:  Routing - potentially group these
+  '/:context(all)/:streamType(stream)':                                   `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  '/:context(all)/:streamType(discussions)':                              `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  '/:context(all)/:streamType(events)':                                   `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  '/:context(all)/:streamType(projects)':                                 `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  '/:context(all)/:streamType(proposals)':                                `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  '/:context(all)/:streamType(requests-and-offers)':                      `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  '/:context(all)/:streamType(resources)':                                `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+
+  // User-Specific Routes
+  '/:context(my)/announcements':                                          `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  '/:context(my)/groups':                                                 `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/User Groups`,
+  '/:context(my)/interactions':                                           `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  '/:context(my)/mentions':                                               `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Home Tab/Stream`,
+  '/:context(my)/account':                                                `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Settings Tab`,
+  '/:context(my)/blocked-users':                                          `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Settings Tab`,
+  '/:context(my)/edit-profile':                                           `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Settings Tab`,
+  '/:context(my)/invitations':                                            `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Settings Tab`,
+  '/:context(my)/locale':                                                 `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Settings Tab`,
+  '/:context(my)/saved-searches':                                         `${AUTH_ROOT_SCREEN_NAME}/Drawer/Tabs/Settings Tab`,
+  // TODO: Routing - this used to go to Notification within the Home Tab, which I think may still pull-up a modal.
```

Comment:
/my/notifications is the notifications settings page, not the notifications themselves

Reply:
Sorry, that and many things here are already corrected. If you have a minute let's live sync? It'll go much quicker and all good things.